viz/charts/linkerd-viz/values.yaml

```diff
@@ -50,6 +50,9 @@ tolerations: &default_tolerations
 # Enable this only when you have multiple replicas of components.
 enablePodAntiAffinity: false
 
+# -- enables the creation of pod disruption budgets for tap and tap-injector components
```

Comment:
Can you please add `web` to the list, and re-run `bin/helm-docs`?

Reply:
done. also added metrics-api.